.drone.sh

```diff
@@ -12,4 +12,4 @@ fi
 echo "Building $VERSION"
 
```

Comment:
@laszlocph do we use this script somewhere ? `rg` was not able to find a ref in this codebase

Reply:
Not anymore